### ISSUE TITLE
compiler style warning: inconsistent-missing-override

### ISSUE THREAD

**Issue by null77** [KhronosGroup/Vulkan-ValidationLayers#2419]
Example failure:

```
../../third_party/vulkan-deps/vulkan-validation-layers/src/layers\stateless_validation.h(1512,10): error: 'PreCallValidateGetDeviceGroupSurfacePresentModes2EXT' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    bool PreCallValidateGetDeviceGroupSurfacePresentModes2EXT(VkDevice device, const VkPhysicalDeviceSurfaceInfo2KHR *pSurfaceInfo,
         ^
../../third_party/vulkan-deps/vulkan-validation-layers/src/layers/generated\chassis.h(4412,22): note: overridden virtual function is here
        virtual bool PreCallValidateGetDeviceGroupSurfacePresentModes2EXT(VkDevice device, const VkPhysicalDeviceSurfaceInfo2KHR* pSurfaceInfo, VkDeviceGroupPresentModeFlagsKHR* pModes) const { return false; };
                     ^
1 error generated.
```

This is popping up in the Chromium integration. We turn on a lot of additional compiler warnings and use -Werror. I'll submit a CL to disable this specific warning in GN. If the VVL maintainers would like to enforce this style that would involve fixing the code generation and turning this warning on in your presubmit.

Context: https://chromium-review.googlesource.com/c/chromium/src/+/2595260

**Comment by mark-lunarg** [comment]
@null77, is this the only one of these you get?  I haven't confirmed, but my impression is that the override keyword is probably just missing from that one line in stateless_validation.h.  A couple of weeks ago we did update the code-gen for this and the warning is enabled on the Android compilers, but this particular case is Windows-only (possibly explaining why it was missed).  If this is actually the case, it might be simpler to add the keyword to the spot where it's missing?

**Comment by null77** [comment]
Hey Mark, if you want to fix the warning, please go ahead and make sure you turn this on as an error in your presubmit. If it's not blocking presubmits this is just going to pop up again and I'd rather we just turn it off permanently. i.e. either you integrate it into your process or we just ignore it.

**Comment by mark-lunarg** [comment]
It is integrated into our process and it does block submissions on failure, but the compiler we use for this doesn't build Windows. I didn't think MSVC supported this warning, but I'll go back and check.

**Comment by null77** [comment]
Ah, I get it. You don't have coverage for Windows/Clang. I didn't see a warning for a missing override in MSVC when I looked quickly. Not sure if you have the capacity to add a build config for win/clang, but if you think this breakage is fairly rare then please go ahead and land the warning fix instead of this CL.
